### ISSUE TITLE
What is GitHub

### ISSUE THREAD

**Issue by teremuhamblin** [teremuhamblin/about-me#2]
Git is version management software. Git will allow us to save the various modifications made to a project and to be able to return to a previous version of the project.

In the language of version management systems, the copy of all the files in a project and their version located on the central server is called a repository. Git also calls this “repository” or “repo” for short.

GitHub is an online service that allows you to host Git repositories or repositories. It is the largest Git repository host in the world.

Many of the repositories hosted on GitHub are public, meaning that anyone can download the code from these repositories and contribute to their development by proposing new features.

To recap, and to be clear on this point: Git is version management software while GitHub is an online Git repository hosting service that acts as a central server for these repositories.
